### PR TITLE
#580 - Wagtail tweaks (underline, etc.)

### PR DESCRIPTION
**Pull request by blms**
## What this PR does

- Per #580:
  - Adds underline using `mark` elements to Wagtail editor toolbar
  - Removes embed option (until/unless we have a design for it)
  - Sets new content pages to appear in menu by default
  - Fixes bug with back-to-back blockquotes

## dev review notes

- My main question: how should I test `wagtail_hooks.py`? I put in a preliminary test just to get some coverage but I'm not really sure how to approach such a test. Ideally I'd like to test that it actually outputs a `<mark class="underline">` element, but I don't know if that's possible given all the Wagtail library code.

### CODE REVIEW

File: geniza/pages/models.py [hunk 1]
```diff
@@ -16,11 +16,11 @@ class HomePage(Page):
             "h3",
             "bold",
             "italic",
+            "underline",
```
Comment (rlskoeser) [inline geniza/pages/models.py:19]: Are these two lists of features the same (for home page and content page)? can we define them once and reuse?

**Review by rlskoeser — APPROVED** [review]
Looks good. Cool to see the custom wagtail/draftail feature, I don't think we've used that much (if at all) yet on other projects where we're using wagtail.

File: geniza/pages/models.py (hunk 1 above)
Comment (blms) [reply geniza/pages/models.py:19]: Yes! Actually will be doing that in work on #581, so I'll go ahead and merge this first.

**Comment by rlskoeser** [comment]
> * My main question: how should I test `wagtail_hooks.py`? I put in a preliminary test just to get some coverage but I'm not really sure how to approach such a test. Ideally I'd like to test that it actually outputs a `<mark class="underline">` element, but I don't know if that's possible given all the Wagtail library code.

My approach is that we only need to test our own code, not third-party code (unless we have reason to, e.g. if we know it doesn't behave correctly in certain scenarios). It's not obvious that there's a very meaningful way to test the functionality related to this hook... Any idea, would your current test alert us if this hook didn't work with a newer version of wagtail?

**Comment by blms** [comment]
> Any idea, would your current test alert us if this hook didn't work with a newer version of wagtail?

@rlskoeser It might! At the very least, it will alert us if the API changes to the point that this function produces any exceptions.

**Comment by rlskoeser** [comment]
> > Any idea, would your current test alert us if this hook didn't work with a newer version of wagtail?
> 
> @rlskoeser It might! At the very least, it will alert us if the API changes to the point that this function produces any exceptions.

yes, I was thinking that too — if it registers successfully then that might be enough for what we need in terms of testing/maintenance

**Pull request merged**